### PR TITLE
K8s: change pvc warning to link to new feature

### PR DESCRIPTION
**Pull request by kaitlynmichael**

### CODE REVIEW

File: content/operate/kubernetes/recommendations/persistent-volumes.md [hunk 1]
```diff
@@ -49,9 +49,7 @@ requirements]({{< relref "/operate/rs/installing-upgrading/install/plan-deployme
 To explicitly specify the persistent storage size, use the *volumeSize*
 property as described in the example above.
 
-{{< warning >}}
-Be aware the persistent volume size cannot be changed after deployment. Trying to change this value after deployment could result in unexpected and potentially damaging behavior. Please be sure your specified *volumeSize* is correct at the time of creation.
-{{< /warning >}}
+Persistent volume claims can be expanded, but not reduced after creation. See [Expand PersistentVolumeClaim (PVC)]({{relref "/operate/kubernetes/re-clusters/expand-pvc"}}) for details.
```
Comment (dwdougherty) [inline content/operate/kubernetes/recommendations/persistent-volumes.md:52]: Link doesn't display.

```suggestion
Persistent volume claims can be expanded, but not reduced after creation. See [Expand PersistentVolumeClaim (PVC)]({< {relref "/operate/kubernetes/re-clusters/expand-pvc" >}}) for details.
```

File: content/operate/kubernetes/recommendations/sizing-on-kubernetes.md [hunk 2]
```diff
@@ -114,6 +114,8 @@ Then, apply the file by running:
     kubectl apply -f redis-enterprise-cluster.yaml
 
 {{< warning >}}
-Be aware that [persistent volume size]({{< relref "/operate/kubernetes/recommendations/persistent-volumes.md" >}}) cannot be changed after deployment. When adjusting compute resources, make sure the ratio of persistent volume size and the new memory size are in accordance to the [Hardware
-requirements]({{< relref "/operate/rs/installing-upgrading/install/plan-deployment/hardware-requirements.md" >}}).
+When adjusting compute resources, make sure the ratio of persistent volume size and the new memory size are in accordance to the [Hardware
+requirements]({{< relref "/operate/rs/installing-upgrading/install/plan-deployment/hardware-requirements.md" >}}). 
+
+Persistent volume claims can be expanded, but not reduced after creation. See [Expand PersistentVolumeClaim (PVC)]({{relref "/operate/kubernetes/re-clusters/expand-pvc"}}) for details.
```
Comment (dwdougherty) [inline content/operate/kubernetes/recommendations/sizing-on-kubernetes.md:120]:
```suggestion
Persistent volume claims can be expanded, but not reduced after creation. See [Expand PersistentVolumeClaim (PVC)]({{< relref "/operate/kubernetes/re-clusters/expand-pvc" >}}) for details.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
A couple of link issues. See suggestions.

File: content/operate/kubernetes/recommendations/persistent-volumes.md (hunk 1 above)
Comment (kaitlynmichael) [inline content/operate/kubernetes/recommendations/persistent-volumes.md:52]:
```suggestion
Persistent volume claims can be expanded, but not reduced after creation. See [Expand PersistentVolumeClaim (PVC)]({{<relref "/operate/kubernetes/re-clusters/expand-pvc">}}) for details.
```

**Pull request merged**